### ISSUE TITLE
Packaging request: Enable libseccomp python functionality

### ISSUE THREAD

**Issue by deliciouslytyped** [NixOS/nixpkgs#99553]
https://github.com/seccomp/libseccomp appears to be packaged at https://github.com/seccomp/libseccomp , however https://github.com/seccomp/libseccomp/tree/master/src/python seems to contain a python module that is currently not built.
libseccomp is packaged at https://github.com/NixOS/nixpkgs/pull/23303#issuecomment-284244324

cc @thoughtpolice @fpletz @c0bw3b because https://github.com/NixOS/nixpkgs/pull/23303#issuecomment-284244324

**Comment by deliciouslytyped** [comment]
Check the configure script output, looks like it just needs cython (and python?) in scope to build the python component.

**Comment by deliciouslytyped** [comment]
It's dirty but its a POC.

--enable-python seems to be needed to enable the python module
--enable-static seems to be needed to get a .a file otherwise it errors in the python with:

```
Making all in python
  PYTHON   build
gcc: error: ../.libs/libseccomp.a: No such file or directory
```

I didn't look for any documentation or explanations on how this is supposed to be done.

```
VER=20.03 PYTHONPATH=$(nix-build --no-out-link -I nixpkgs=channel:nixos-$VER -E "with import <nixpkgs> {}; libseccomp.overrideAttrs (old: { buildInputs = old.buildInputs ++ [ python37Packages.cython ]; configureFlags = (old.configureFlags or []) ++ [ ''--enable-python'' ''--enable-static'' ]; })")/lib/python3.7/site-packages/; \
 nix-shell -I nixpkgs=channel:nixos-$VER -p python37 --run "PYTHONPATH=$PYTHONPATH python"
```

```
Python 3.7.6 (default, Dec 18 2019, 19:23:55) 
[GCC 9.2.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import seccomp
>>> dir(seccomp)
['ALLOW', 'Arch', 'Arg', 'Attr', 'EQ', 'ERRNO', 'GE', 'GT', 'KILL', 'KILL_PROCESS', 'LE', 'LOG', 'LT', 'MASKED_EQ', 'NE', 'SyscallFilter', 'TRACE', 'TRAP', '__author__', '__builtins__', '__date__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__pyx_unpickle_Attr', '__spec__', '__test__', 'c_str', 'errno', 'get_api', 'resolve_syscall', 'set_api', 'system_arch']
```

**Comment by thoughtpolice** [comment]
Fixed by #103952 (which must go to staging, and then master)
